### DISCUSSION TITLE
Is it possible to select outputs for the vtk file ?

### DISCUSSION THREAD

**Discussion by AlexSacMorane** [idaholab/moose#23067] [Q&A Getting Started] [answered]
Dear all,

I am working with the phase-field module of Moose. I need to work on specific data from the output file. I have a python script to automatize the file reading, so I use a VTK file.

The file is large, and I do not use most of it. In fact, I need only etai, c, Ed_mec and Ed_prec. Is there a way to filter the VTK file writing to reduce the size?
 
I know I can reduce the size by commenting lines about the [GlobalParams] part. But Ed_mec and Ed_prec are not some variables, that is why I print all the data. 

My .i file and some vtk outputs are in the attachments.
I stay available
Thanks in advance

Alexandre SM

[Select_Outputs.zip](https://github.com/idaholab/moose/files/10345808/Select_Outputs.zip)

**Comment by GiudGiud** [comment, marked as answer]
Hello

In the Outputs block you can specify which fields you want to include or hide using the hide and show parameters

Guillaume

**Reply by AlexSacMorane** [reply]
Thank for your answer
